lang/go/eval/symbol/series.go

```diff
@@ -28,8 +30,32 @@ type SeriesSymbol struct {
 	Elem  Symbols     `ko:"name=elem"`
 }
 
-var _ Symbol = &SeriesSymbol{}
+var (
+	_            Symbol = &SeriesSymbol{}
+	seriesGoType        = reflect.TypeOf([]interface{}{})
+)
+
+// DisassembleToGo converts a Ko value into a Go value
+func (ss *SeriesSymbol) DisassembleToGo(span *model.Span) (reflect.Value, error) {
+	filtered := FilterEmptySymbols(ss.Elem)
+	elements := make([]reflect.Value, 0, len(filtered))
```

Comment:
You don't need to make two slices (elements and slice, below). Use could use the for loop to fill in 'slice'.